lib/ansible/modules/database/postgresql/postgresql_user.py

```diff
@@ -725,7 +734,7 @@ def main():
             port=dict(default='5432'),
             fail_on_user=dict(type='bool', default='yes'),
             role_attr_flags=dict(default=''),
-            encrypted=dict(type='bool', default='no'),
+            encrypted=dict(type='bool', default='yes'),
```

Comment:
I don't see any check which set encrypted for PostgreSQL 10 and above.

Reply:
Sorry for not being obvious in the PR explanation/git commit message, but I didn't do that, I made it the default and here is the reasoning from the PR explanation:
>Starting with Postgres 10 UNENCRYPTED passwords are removed and because of that this module fails with the default encrypted=no. Also encrypted passwords are supported since version 7.2
(https://www.postgresql.org/docs/7.2/static/sql-createuser.html) which went EOL in 2007 and since 7.3 they are the default. Because of this it makes a lot more sense to make encrypted=yes the default. This won't break backward compatibility, the module would just update the user's password in the DB in the hashed format and everything else will work like before.

I was thinking to make it check for the version, but because of what I wrote above and because this change won't affect the users in any way except the module would report once as changed to update the password in the DB in a hashed form, nothing more, everything else would work the same. The other reason why I think encrypted should be the default is because it would make the whole `encrypted` parameter complicated without a reason (`no` before 10, `yes` after). Also because storing passwords in plain text format is not a recommended way of doing it anyway. All Postgres versions from 7.3 store user passwords as hashed and the module should follow that practice as well.

Only users that could have issues are users of versions <7.2, but do we really need to support extremely old Postgres versions which are EOL for more than 12 years now.